client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCache.java

```diff
@@ -10,11 +10,11 @@
 import org.infinispan.client.hotrod.jmx.RemoteCacheClientStatisticsMXBean;
 import org.infinispan.commons.api.BasicCache;
 import org.infinispan.commons.api.TransactionalCache;
-import org.infinispan.commons.api.query.Query;
 import org.infinispan.commons.util.CloseableIterator;
 import org.infinispan.commons.util.CloseableIteratorCollection;
 import org.infinispan.commons.util.CloseableIteratorSet;
 import org.infinispan.commons.util.IntSet;
+import org.infinispan.query.dsl.Query;
 import org.reactivestreams.Publisher;
```

Comment:
I guess this change in the interface won't break compatibility somehow?

Reply:
Let me look at this closer.

Reply:
Ah, it was just a conflict https://github.com/infinispan/infinispan/commit/54a9574e5c597ba69af8382c89ea2b2baa3cd53c. Reverting it.